src/deadline/houdini_submitter/otls/deadline_cloud.hda/Driver_1deadline__cloud/DialogScript

```diff
@@ -589,6 +591,7 @@
                 default { "50" }
                 range   { 1 100 }
                 parmtag { "script_callback_language" "python" }
+                help    "The job's scheduling priority. The numerically largest priority is scheduled first. When two jobs have the same priority, the oldest job is scheduled first."
```

Comment:
Hmm, without getting too into the weeds here, is it possible to link to service docs? Might not be possible since it's hover text. Maybe we just direct them to the service text to learn more about how priorities are resolved rather than here?

ie.

```suggestion
                help    "The job's scheduling priority. Larger priorities are given preference to smaller priorities. See AWS Deadline Cloud service documentation for more info."
```

Reply:
Unfortunately links don't work as it is hover text, but we can reference the service documentation!